fhir-tools/src/main/java/com/ibm/fhir/tools/CodeGenerator.java

```diff
@@ -3265,13 +3263,27 @@ public void generateCodeSubtypeClass(String bindingName, String valueSet, String
             ._super(args("builder"))
         .end().newLine();
 
+        cb.javadocStart()
+            .javadoc("Get the value of this " + bindingName + " as an enum constant.")
+            .javadoc("@deprecated replaced by {@link #getValueConstant()}")
+            .javadocEnd();
+        cb.annotation("Deprecated");
         cb.method(mods("public"), "ValueSet", "getValueAsEnumConstant")
             ._return("(value != null) ? ValueSet.from(value) : null")
         .end().newLine();
 
+        cb.javadocStart()
+            .javadoc("Get the value of this " + bindingName + " as an enum constant.")
+            .javadocEnd();
+        cb.method(mods("public"), "Value", "getValueConstant")
```

Comment:
Anyone with strong feelings on the new name for the `getValueAsEnumConstant` method?

John and I were between `getValueAsEnum` and `getValueConstant` with him preferring the former and me preferring the latter.

Reply:
No preference, I think I like the getValueAsEnum, however by comment it's very clear it's a enum and a constant.   both seem fine to me.

Reply:
thanks paul.  thats sort of where john landed too.  will merge this one as-is unless someone objects.

Reply:
I'm with John.